src/SIS.HTTP/Cookies/IHttpCookieCollection.cs

```diff
@@ -13,5 +13,7 @@ public interface IHttpCookieCollection : IEnumerable<HttpCookie>
         HttpCookie GetCookie(string key);
 
         bool HasCookies();
+		
+		bool IfCookieIsNew(string key, string value);
```

Comment:
A different method name such as "IsCookieNew" may be better than "IfCookieIsNew"